src/main/java/MainWindow.java

```diff
@@ -0,0 +1,53 @@
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
```

Comment:
Maybe it would have been better to leave a line between your import statements and your javadocs?